wandb/internal/update.py

```diff
@@ -63,3 +74,12 @@ def check_available(current_version):
             " --pre" if pip_prerelease else "",
         )
     )
+
+    yank_message = None
+    if deleted:
+        yank_message = "WARNING: your current version has been DELETED"
+    elif yanked:
+        yank_message = "WARNING: your current version has been YANKED"
+
+    # A new version is available!
+    return upgrade_message, yank_message
```

Comment:
lets just have a separate deleted message.  that makes it easier to do things differently in the future.